src/browser.js

```diff
@@ -2,18 +2,22 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
  */
 
 /* eslint-env browser */
 import React from 'react';
 import {createPlugin} from 'fusion-core';
+import type {FusionPlugin} from 'fusion-core';
 
 import {Provider as StyletronProvider} from 'styletron-react';
 import {Client as Styletron} from 'styletron-engine-atomic';
 
 let engine;
 
-export default __BROWSER__ &&
+const plugin =
+  __BROWSER__ &&
```

Comment:
Hmm, might need to test this, but I thought we would be able to remove these fences now that we have landed the babel/uglify plugin stuff.